resources/views/components/general/encapsulated/transaction-type-badge.blade.php

```diff
@@ -0,0 +1,5 @@
+@props(['transaction'])
+
+<div {{ $attributes->class('text-xs font-semibold rounded border border-transparent dark:bg-transparent encapsulated-transaction-type px-[3px] py-[2px] bg-theme-secondary-200 leading-[15px] dark:border-theme-secondary-800 dark:text-theme-secondary-500') }}>
```

Comment:
```suggestion
<div {{ $attributes->class('text-xs font-semibold rounded border border-transparent dark:bg-transparent encapsulated-transaction-type px-[3px] py-0.5 bg-theme-secondary-200 leading-[15px] dark:border-theme-secondary-800 dark:text-theme-secondary-500') }}>
```

also aren't most `leading-[17px]` instead of 15px? 🤔

Reply:
see slacky